fuzzers/utils.py

```diff
@@ -162,6 +162,10 @@ def set_default_optimization_flag(env=None):
 
 def initialize_flags(env=None):
     """Set initial flags before fuzzer.build() is called."""
+    for flag_var in ['CFLAGS', 'CXXFLAGS']:
```

Comment:
Consider leaving a comment explaining why we are deleting these env vars here

Reply:
Am I actually not sure we need to delete them, so I got rid of these lines.